script/update_containers.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -e -u
+
+SCRIPT_NAME=$(basename "$0")
+
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME <prod|beta|test> [on <hosts list>]"
+
+Update MusicBrainz website/webservice containers on specified hosts.
+If no (space-delimited) hosts list is specified, update on all hosts.
+EOH
+)
+
+MB_SERVER_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)
+
+cd "$MB_SERVER_ROOT"
+
+if [ $# -eq 0 ]
+then
+  echo >&2 "$SCRIPT_NAME: missing arguments"
+  echo >&2 "$HELP"
+  exit 64
+elif echo "$1" | grep -Eqvx 'prod|beta|test'
+then
+  echo >&2 "$SCRIPT_NAME: unrecognized argument: $1"
+  echo >&2 "$HELP"
+  exit 64
+fi
+
+DEPLOY_ENV=$1
+shift
+
+SERVICES="musicbrainz-webservice musicbrainz-website"
+
+if [ $# -eq 0 ]
+then
+  LIST_METABRAINZ_HOSTS=${LIST_METABRAINZ_HOSTS:-../docker-server-configs/scripts/list_nodes.sh}
+  if ! type "$LIST_METABRAINZ_HOSTS" >/dev/null
+  then
+    sed -E 's/^ *> ?//' >&2 << ....EOM
+    > $SCRIPT_NAME: cannot list hosts per service/deploy env
+    >
+    > Please set \$LIST_METABRAINZ_HOSTS or specify hosts list
+....EOM
+    exit 69
+  fi
+
+  HOSTS=$(
+    for service in $SERVICES
+    do
+      "$LIST_METABRAINZ_HOSTS" "$service" "$DEPLOY_ENV"
+    done | sort -u
+  )
+else
+  if [ "$1" != 'on' ]
+  then
+    echo >&2 "$SCRIPT_NAME: missing 'on' separator"
```

Comment:
Not sure why it enforces pseudo English here, TBH. Can we keep it simple like `update_containers.sh <prod|beta|test> [<hosts list>]`?

Reply:
Deploy environment was initially facultative, that was supposed to help with options parsing. It is indeed no longer useful.

Reply:
Removed in https://github.com/metabrainz/musicbrainz-server/pull/1337/commits/391f8f42d23edc78cb04f73265774cea47a9c82d